pkgs/racket-doc/scribblings/reference/port-lib.scrbl

```diff
@@ -25,41 +25,52 @@ on @racket[in] until it produces @racket[eof].
 (port->list read-number (open-input-string "12345"))
 ]}
 
-@defproc[(port->string [in input-port? (current-input-port)]) string?]{
+@defproc[(port->string [in input-port? (current-input-port)]
+                       [#:close? close? any/c #f])
+         string?]{
 
 Reads all characters from @racket[in] and returns them as a string.
+When @racket[close?] is @racket[#t], the input port is closed.
```

Comment:
Should these say `true` instead of `@racket[#t]`?

Most places I looked in the docs say "If .... is `#f` .... otherwise ....", but [`copy-directory/files`](http://docs.racket-lang.org/reference/Filesystem.html#%28def._%28%28lib._racket%2Ffile..rkt%29._copy-directory%2Ffiles%29%29) says `true`